voila/exporter.py

```diff
@@ -16,6 +16,11 @@
 except ImportError:
     from jinja2 import contextfilter as pass_context
 
+try:
+    from mistune import Renderer as MistuneRenderer
```

Comment:
Maybe we can add a quick comment to specify which version of `mistune` requires this import?

So it can be removed at some point later when the minimum of `mistune` exports `HTMLRenderer`.

Reply:
Actually we should probably require `markdown_renderer_class` to be: 
```python
    markdown_renderer_class = traitlets.Type(VoilaMarkdownRenderer).tag(config=True)
```

Because the `markdown_renderer_class` class must have a constructor which accepts a `contents_manager` as parameter. This would have a side effect that we don't need to import anything from Mistune.

Reply:
Nice, even better.